Python/Product/Common/Strings.resx

```diff
@@ -2257,28 +2257,28 @@ Expand selection to the full expression?</value>
     <value>Debugger package could not be loaded</value>
   </data>
   <data name="PtvsdDisableCaption" xml:space="preserve">
-    <value>Disable the experimental debugger</value>
+    <value>Enable the legacy debugger</value>
   </data>
   <data name="PtvsdDisableSubtext" xml:space="preserve">
-    <value>Future debugging sessions will not use the experimental debugger. You can also change this setting in the Tools/Options/Python/Experimental page.</value>
+    <value>Future debugging sessions will use the legacy debugger. You can change this setting back in Tools &gt; Options &gt; Python &gt; Debugging page.</value>
   </data>
   <data name="PtvsdLearnMoreCaption" xml:space="preserve">
     <value>Learn more</value>
   </data>
   <data name="PtvsdLearnMoreSubtext" xml:space="preserve">
-    <value>View documentation in your web browser on supported Python environments and how to upgrade or uninstall the ptvsd debugger package.</value>
+    <value>View documentation on supported Python environments and how to upgrade or uninstall the ptvsd debugger package.</value>
   </data>
   <data name="PtvsdIncompatibleEnvTitle" xml:space="preserve">
-    <value>Debugger is incompatible with this environment</value>
+    <value>Debugger does not support this Python environment</value>
   </data>
   <data name="PtvsdIncompatibleEnvMessage" xml:space="preserve">
-    <value>The experimental debugger is incompatible with this Python environment. See the documentation for a list of supported Python environments.</value>
+    <value>To debug this Python environment use the legacy debugger instead. It is likely using an older version of Python, see the documentation for a list of supported Python versions and environment types.</value>
   </data>
   <data name="DisableExperimentalDebugger" xml:space="preserve">
-    <value>Disable the experimental debugger</value>
+    <value>Enable the legacy debugger</value>
```

Comment:
"Use" makes more sense than "Enable" alongside a checkbox.

Reply:
Oh wait, this is going in the task dialog too, right?

Reply:
Right, it's a task dialog button, not a check box. Also it's not immediate, the current debugging session that was just started is not affected.

Reply:
Oh, there's two of those identical strings, and this one is not used, will delete.